main.html

```diff
@@ -38,6 +41,108 @@ <h2 class="proffesion">Full Stack Dev</h2>
+            <a href="https://github.com/Flunavgt"><img class="link" src="pictures/github.png" alt=""></a>
+            <a href="https://twitter.com/?lang=en"><img class="link" src="pictures/twitter.png" alt=""></a>
+        </div>
+    </section>
+    <section>
+        <div class="workguide">
+            <i class="fa fa-arrow-down" aria-hidden="true"></i>
+            <p class="work">Work</p> 
+        </div>
+        <p class="worktitle">My Recent Work</p>
+        <img class="separator" src="./pictures/vector4.png" alt="line to separate information">
+        <div class="worklist">
+            <div class="work1">
+                <img src="./pictures/myproject.jpg" alt="fill image while coding" class="worksample1">
+                <p class="work1title">Multi-Post Stories</p>
+                <p class="workdescription">A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. Has been the industry standard dummy 
+                    text ever since the 1500s, when an unknown printer took standard dummy test.
+                </p>
+                <ul class="usedcode">
+                    <li>css</li>
+                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li>html</li>
+                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li>bootstrap</li>
+                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li>Ruby</li>
+                </ul>
+                    <button type="button" onclick="location.href='https://www.google.com/'">See Project</button>
+            </div>
+            <div class="work2">
+                <img src="./pictures/myproject.jpg" alt="fill image while coding" class="worksample1">
+                <p class="work1title">Multi-Post Stories</p>
+                <p class="workdescription">A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. Has been the industry standard dummy 
+                    text ever since the 1500s, when an unknown printer took standard dummy test.
+                </p>
+                <ul class="usedcode">
+                    <li>css</li>
+                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li>html</li>
+                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li>bootstrap</li>
+                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li>Ruby</li>
+                </ul>
+                    <button type="button" onclick="location.href='https://www.google.com/'">See Project</button>          
+            </div>
+            <div class="work3">
+                <img src="./pictures/myproject.jpg" alt="fill image while coding" class="worksample1">
+                <p class="work1title">Multi-Post Stories</p>
```

Comment:
- [ ] Nice working on your `Work Section`, kindly change the `work title` paragraph `p` tag to a `header hx` tag for better use of HTML Semantics.